code/modules/mob/living/simple_animal/bees/bees_mob.dm

```diff
@@ -194,6 +194,18 @@
 	if (src)
 		qdel(src)
 
+/mob/living/simple_animal/bee/reagent_act(id, method, volume)
+	if(isDead())
+		return
+
+	.=..()
+
+	switch(id)
+		if(TOXIN)
+			visible_message("<span class='danger'>The bees stop moving...</span>")
```

Comment:
This seems weird to me.
If all the bees die this message shouldn't be necessary because presumably death() does it (?)
If not all the bees die `panic_attack` is called, and you say they will get mad at everyone, which makes me think they'd start doing the opposite of "stop moving".
What's the matter?

Reply:
Several bees (four to eleven) stop moving is what that message means. I thought it would be confusing if there was no feedback message.

When you kill bees, death() is called on a bee **datum** but killing the last bee in a stack calls **qdel, not death** for the bee mob.

Reply:
🤷‍♂️